### DISCUSSION TITLE
[Docs] 文档上的示例代码全都是ts版本的，但是用脚手架创建项目的时候ts可选可不选，示例代码将来会有一个js版本可供选择吗？v3+eleplus 有不少人不选ts

### DISCUSSION THREAD

**Discussion by huangwenboha563** [element-plus/element-plus#10198] [Feature Request]
Feature type: **`Docs`**

## Details

- Enhanced component name: `All`

## Description

做vue3+elementplus的项目的时候没有选择ts，但是官网示例全都是ts版本的示例。代码复制下来之后还要手动去删调类型...

### Additional comments

感谢

**Comment by huangwenboha563** [comment]
示例代码支持js版本吗

**Comment by huangwenboha563** [comment]
![image](https://user-images.githubusercontent.com/24206806/197437133-7eadb23f-1256-47f0-99fb-27fd9663d2b3.png)
like this

**Comment by huangwenboha563** [comment]
switch javascript or switch typescript

**Comment by dxygit1** [comment]
![image](https://github.com/element-plus/element-plus/assets/57128304/5c098153-bd08-4902-b3f6-d9ceb144510d)

**Comment by ZhiZunJava** [comment]
ts:带了类型的js

**Comment by tyj-321** [comment]
ant design的这个支持js和ts切换，我认为是一个非常好的特性

**Comment by NidhoggDJoking** [comment]
![微信截图_20240717092635](https://github.com/user-attachments/assets/b5530d80-b5d5-44fd-94c9-7a0c7c5c525e)

类似于 ant design 模式 或者参照Vue官网来一个总开关控制

![微信截图_20240717092849](https://github.com/user-attachments/assets/bc0d1718-b36f-44b3-9cdc-b5efca7d817d)
